patient-appointment-management/README.md

```diff
@@ -0,0 +1,61 @@
+# patient-appointment-management
+
+Patient Appointment Management including notification and reminders sent via SMS
+
+## Pre-requisites
+
+- AWS Account with credentials to an admin-level IAM user
```

Comment:
From the values in `.env`, this seems to require an AWS appointments service user as well.

Reply:
The 'DEPLOYER ...' IAM user will need to pre-created by the customer in order to deploy the CF stack. This is being documented in the customer IG.

Reply:
AWS AdministratorAccess policy is not truly required. However, the CloudFormation template creates IAM roles, policies, and various resources. The specific permissions that the CF will need can be created, but then the customer will have to pre-create the role with the permission somehow. The CodeExchange app is blue print for customer so we are recommending that they create a separate AWS account deploy this.